doc/source/parameters/cfd/laser_heat_source.rst

```diff
@@ -0,0 +1,59 @@
+Laser heat source
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If a laser heat source is present in a simulation, it can be added in this section. The default parameters are:
+
+.. code-block:: text
+
+   subsection laser parameters
+	set enable                 = false
+	set concentration factor   = 2.0
+	set power                  = 100.0
+	set absorptivity           = 0.5
+	set penetration depth      = 0.0
+	set beam radius            = 0.0
+	set start time             = 0.0
+	set end time               = 1.0
+	set beam orientation       = z
+	subsection path
+	  set Function expression  =  0.0; 0.0
+	end
+   end
+
+* The ``enable`` parameter is set to true if the problem has a laser heat source term and enables its calculation.
+
+* Laser ``concentration factor`` parameter indicates the definition of the beam radius. In almost all the articles, it is assumed equal to 2.0.
+
+* The ``power`` parameter sets the power of the laser in W.
```

Comment:
I don't think the problem will be in W if the distance unit used is not in meters. maybe give a more general form as L^2MT^-3